docs/sdks/sdk-ref-python.md

```diff
@@ -221,6 +222,161 @@ There are two operations that apply to Advertisers/Data Providers:
 
 ### Map DII to Raw UID2s
 
+1. Create an IdentityMapV3Client as an instance variable:
+   ```py
+    identity_map_v3_client = IdentityMapV3Client(UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY)
+   ```
+
+2. Create an IdentityMapV3Input object. You can use emails, phone numbers, or their hashed forms:
+   ```py
+   input = IdentityMapV3Input.from_emails(["user@example.com", "user2@example.com"])
+   ```
+   Or combine multiple identity types:
+      ```py
+      input = IdentityMapV3Input()
+          .with_email("user@example.com")
+          .with_phone("+12345678901")
+          .with_hashed_email("pre_hashed_email")
+          .with_hashed_phone("pre_hashed_phone")
+      ```
+
+3. Call a function that takes the input and generates an IdentityMapV3Response object:
```

Comment:
`Call a function that takes the input` sounds redundant. Maybe just name the function?